src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -611,15 +611,27 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt, $
             }
 
             if ($token->equals('(')) {
-                $index = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
+                $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $index);
 
-                continue;
+                if (false !== strpos($tokens->generatePartialCode($index, $end), 'function (')) {
+                    continue;
```

Comment:
Code could be simplified by inverting the condition of the `if` and keeping `continue` after it, removing the need for an `else` branch. Same below.